clients/git/client.go

```diff
@@ -37,6 +39,7 @@ const repoDir = "repo*"
 var (
 	errNilCommitFound = errors.New("nil commit found")
 	errEmptyQuery     = errors.New("query is empty")
+	errNotImplemented = errors.New("not implemented")
```

Comment:
our other clients return `clients.ErrUnsupportedFeature`. Let's use that here and be consistent

Reply:
👍